treeherder/etl/perf.py

```diff
@@ -151,6 +153,8 @@ def _load_perf_datum(job, perf_datum):
                     # null (None). Null indicates no preference has been
                     # set.
                     'should_alert': subtest.get('shouldAlert'),
+                    'alert_change_type': PerformanceSignature._get_alert_change_type(
+                        suite.get('alertChangeType')),
```

Comment:
Should this be subtest.get? Or are you using the setting in suite here intentionally to force all subtests to have the same setting as the suite they belong to? In which case is there a point in having the 'alertChangeType' setting in the performance-artifact.json?